stage/tests/lms/test_dashboard.py

```diff
@@ -0,0 +1,26 @@
+from bok_choy.web_app_test import WebAppTest
+from ...pages.lms.login_lms import LmsLogin
+from ...pages.lms.dashboard_lms import DashboardPageExtended
+from ...pages.lms.course_page_lms import CourseInfoPageExtended
+from edxapp_pages.lms.courseware import CoursewarePage
+import os
+
+class DashboardTest(WebAppTest):
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
```

Comment:
This is good for now but we may need to put it into a common place, like a base class. No change needed for now; let's keep track of our vars though and plan to move them later.